app/src/main/resources/score-tracker.js

```diff
@@ -0,0 +1,735 @@
+import * as Widgets from './khan-widgets.js';
```

Comment:
Needs some comment at the top of this script explaining what this is, what it does, etc.

Reply:
added comments